clients/ui/docs/local-deployment-guide-ui.md

```diff
@@ -29,9 +29,7 @@ kubectl create namespace kubeflow
 You can now deploy the UI and BFF to your newly created cluster using the kustomize configs in this directory:
 
 ```shell
-cd clients/ui
-
-kubectl apply -k manifests/overlay/standalone -n kubeflow
+kubectl apply -k manifests/kustomize/options/ui/overlays/standalone -n kubeflow
 ```
 
 After a few seconds you should see 2 pods running (1 for BFF and 1 for UI):
```

Comment:
I think this info is outdated, the user should see just 1 pod running

Reply:
Yes, that's right, if we can change it it would be amazing